Android/res/layout/activity_geotag.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <include layout="@layout/actionbar_toolbar"/>
+
+    <View
```

Comment:
@ne0fhyk This view creates the black background that's behind the button at the bottom. It looks similar to an action bar at the bottom